scylla/src/frame/value.rs

```diff
@@ -32,6 +32,7 @@ pub struct Unset;
 pub struct Counter(pub i64);
 
 /// Enum providing a way to represent a value that might be unset
+#[derive(Clone, Copy)]
```

Comment:
contrary to to the commit title, it also adds Copy here, not just Clone. I don't think it's desired, not all Value impls are Copy. I'm actually surprised it compiles - is there some rule in Rust I'm not aware of that would only allow Copy semantics here is V is also Copy?

Reply:
Yes, it works like that. If you add `#[derive(Clone)]` or Copy for a generic struct, it only works when the generic type parameters implement Clone/Copy, too.

Reply:
Ok, it's good then. So the only thing to fix is to mention Copy in the commit message as well.

Reply:
Some source for the above claim: https://doc.rust-lang.org/std/clone/trait.Clone.html#derivable

I'd expect it's similar for Copy.